test/index.ts

```diff
@@ -487,6 +487,31 @@ describe('Bigtable', function() {
         done();
       });
     });
+
+    it('should throw an error if cluster id not provided.', function(done) {
+      const error = new Error(
+        'id of cluster is required to create an instance.'
+      );
+      const instanceOptions = {
+        clusters: [
+          {
+            nodes: 3,
+            location: 'us-central1-f',
+            storage: 'ssd',
+          },
+        ],
+        labels: {'prod-label': 'prod-label'},
+      };
+
+      bigtable.createInstance = function(id, option, callback) {
+        callback(error);
```

Comment:
This test looks to be overriding the behavior of the function we're trying to test. The test should call `bigtable.createInstance` wrapped in a try/catch to assert that the expected error is thrown.

Reply:
Done!!